endpoints/openrtb2/amp_auction_test.go

```diff
@@ -50,6 +50,8 @@ func TestGoodAmpRequests(t *testing.T) {
 	}
 }
 
+// TODO: (@cirla) test `debug=1` query param
```

Comment:
Did you plan to do this one? Same with the other tests.

If not, I think you can just remove the TODO unless you open issues for them... in my experience, untracked TODOs generally do more harm than good.